src/repart/repart.c

```diff
@@ -10065,44 +10065,48 @@ static int resize_backing_fd(
 static int determine_auto_size(
                 Context *c,
                 int level,
-                bool ignore_allocated, /* If true, determines unallocated space needed */
-                uint64_t *ret) {
+                uint64_t *ret_new_size,
+                uint64_t *ret_current_size) {
 
-        uint64_t sum;
+        uint64_t new_size, current_size;
 
         assert(c);
 
-        sum = round_up_size(GPT_METADATA_SIZE, 4096);
+        new_size = round_up_size(GPT_METADATA_SIZE, 4096);
+        current_size = 0;
```

Comment:
Hmm, so `new_size` includes the GPT_METADATA_SIZE, while `current_size` doesn't.